packages/tldraw/src/lib/shapes/text/toolStates/Pointing.ts

```diff
@@ -15,29 +15,50 @@ export class Pointing extends StateNode {
 
 	markId = ''
 
+	enterTime = 0
+	override onEnter(): void {
+		this.enterTime = Date.now()
+	}
+
 	override onExit() {
 		this.editor.setHintingShapes([])
 	}
 
 	override onPointerMove(info: TLPointerEventInfo) {
+		// Create a fixed width shape if the user wants to do that.
+
 		// Don't create a fixed width shape unless the the drag is a little larger,
 		// otherwise you get a vertical column of single characters if you accidentally
 		// drag a bit unintentionally.
+
+		// If the user hasn't been pointing for more than 150ms, don't create a fixed width shape
+		if (Date.now() - this.enterTime < 150) return
+
 		const { editor } = this
-		const { isPointing, originPagePoint, currentPagePoint } = editor.inputs
-		if (
-			isPointing &&
-			Math.abs(originPagePoint.x - currentPagePoint.x) ** 2 >
-				((editor.getInstanceState().isCoarsePointer
-					? editor.options.coarseDragDistanceSquared
-					: editor.options.dragDistanceSquared) *
-					4) / // double the necessary drag distance for text shapes
-					editor.getZoomLevel()
-		) {
+		const { isPointing } = editor.inputs
+
+		if (!isPointing) return
+
+		const { originPagePoint, currentPagePoint } = editor.inputs
+
+		const currentDragDist = Math.abs(originPagePoint.x - currentPagePoint.x)
+
+		const baseMinDragDistForFixedWidth = Math.sqrt(
+			editor.getInstanceState().isCoarsePointer
+				? editor.options.coarseDragDistanceSquared
+				: editor.options.dragDistanceSquared
+		)
+
+		// Ten times the base drag distance for fixed width
+		const minSquaredDragDist = (baseMinDragDistForFixedWidth * 10) / editor.getZoomLevel()
```

Comment:
this makes it take wayyyyy too long to get visual feedback on the drag IMO.

I have no issue with the fact that nobody would want a fixed-width text box smaller than this, but I'd rather have faster feedback while dragging the boxes than a loosely-enforced minimum size like this.

Reply:
it feels better at other zoomed-in, but yeah at 100% it's prbly a touch too long to get the feedback

Reply:
What feels right?

Reply:
mm, personally, feels like 6 as the multiplier might suit better, perhaps 7